wpilibcExamples/src/main/cpp/examples/Gyro/src/Robot.cpp

```diff
@@ -27,8 +27,8 @@ class Robot : public frc::IterativeRobot {
 	}
 
 	/**
-	 * The motor speed is set from the joystick while the RobotDrive turning
-	 * value is assigned from the error between the setpoint and the gyro
+	 * The motor speed is set from the joystick while the DifferentialDrive
+   * turning value is assigned from the error between the setpoint and the gyro
```

Comment:
Did something change from tabs to spaces here?

Reply:
Yep. Will fix.